RTClib.h

```diff
@@ -323,14 +128,16 @@ enum Ds3231Alarm2Mode {
     @brief  RTC based on the DS3231 chip connected via I2C and the Wire library
 */
 /**************************************************************************/
-class RTC_DS3231 {
+class RTC_DS3231 : public RTC_Super {
```

Comment:
Perhaps better left to another pull request? I only split the file into multiple parts because having both the parent and child classes in the same file is a bit silly.

Reply:
It pains me to advocate for multiple files. I dislike long files and multiple files. However I am of the belief a class generally belongs in its own class, assuming the class is independent. I think these driver classes will be independent, however unfortunately probably short. So this could complicate code support. However it may clean things up.

I would also be interested in trying to clean up the shared logic. However if that complicated the interface or performance I guess it should stay the way it is.